locales/de-DE/App.json

```diff
@@ -0,0 +1,247 @@
+  "Alephium doesn't have access to your wallet.\nYou are the only owner.": "Alephium hat keinen Zugriff auf Ihre Brieftasche.\nSie sind der einzige Besitzer.",
+  "Back": "Zurück",
+  "Type your recovery phrase": "Geben Sie Ihre Wiederherstellungsphrase ein",
+  "Error while importing wallet": "Fehler beim Importieren der Brieftasche",
+  "Make sure to store the words in a secure location! They are your wallet's secret recovery phrase.": "Bewahren Sie die Worte an einem sicheren Ort auf! Sie sind die geheime Wiederherstellungsphrase Ihrer Brieftasche.",
+  "All good? Let's continue!": "Alles in Ordnung? Machen wir weiter!",
+  "Everything is ready!": "Alles ist bereit!",
+  "Welcome to Alephium.": "Willkommen in Alephium.",
+  "Let's go!": "Los geht's!",
+  "welcomeScreenOneAddressPerGroupMessage": "welcomeScreenOneAddressPerGroupMessage",
+  "Show advanced options": "Erweiterte Optionen anzeigen",
+  "Hide advanced options": "Erweiterte Optionen ausblenden",
+  "For mining or DeFi use.": "Für Bergbau oder DeFi verwenden.",
+  "Address copied to clipboard!": "Adresse in die Zwischenablage kopiert!",
+  "Your Wallet": "Ihre Brieftasche",
+  "Your address": "Ihre Adresse",
+  "I've copied the words, continue": "Ich habe die Worte kopiert, fahren Sie fort",
+  "Advanced operations reserved to more experienced users. A \"normal\" user should not need to use them very often, if not at all.": "Fortgeschrittene Operationen, die erfahreneren Benutzern vorbehalten sind. Ein \"normaler\" Benutzer sollte sie nicht sehr oft, wenn überhaupt, verwenden müssen.",
+  "It appears that your wallet has too many UTXOs to be able to send this transaction. Please, consolidate (merge) your UTXOs first. This will cost a small fee.": "Es scheint, dass Ihre Brieftasche zu viele UTXOs hat, um diese Transaktion senden zu können. Bitte konsolidieren (fusionieren) Sie zunächst Ihre UTXOs. Dies wird eine kleine Gebühr kosten.",
+  "Welcome!": "Willkommen!",
+  "Welcome back!": "Willkommen zurück!",
+  "Please choose a wallet and enter your password to continue.": "Bitte wählen Sie eine Wallet und geben Sie Ihr Passwort ein, um fortzufahren.",
+  "Create / import a new wallet": "Erstellen / Importieren einer neuen Wallet",
+  "(current)": "(aktuell)",
+  "Default address for sending transactions.": "Standardadresse für das Senden von Transaktionen.",
+  "Error while fetching data for address {{ hash }}": "Fehler beim Abrufen von Daten für Adresse {{ hash }}",
+  "Error while fetching pending transactions for address {{ hash }}": "Fehler beim Abrufen von ausstehenden Transaktionen für Adresse {{ hash }}",
+  "Open in explorer": "Im Explorer öffnen",
+  "Reduce": "Verkleinern:",
+  "Show addresses": "Adressen anzeigen",
+  "More info": "Mehr Infos",
+  "passphraseWarningMessage": "passphraseWarningMessage",
+  "welcomeScreenPassphraseMessage": "welcomeScreenPassphraseMessage",
```

Comment:
These 2 are not translated properly :)

<img width="562" alt="image" src="https://user-images.githubusercontent.com/1579899/197718455-2a59236a-3b23-414c-bfc3-680d4850b90f.png">